components/DarkModeToggle.tsx

```diff
@@ -15,6 +15,20 @@ export default function DarkModeToggle() {
     const newTheme = isDarkMode ? 'light' : 'dark';
     setTheme(newTheme);
     setIsDarkMode(!isDarkMode);
+    const dataTheme = isDarkMode ? 'light' : 'dark';
+    const keyShadow = isDarkMode
```

Comment:
Do we need to define this styles in both files: DarkModeToggle and Layout?

Reply:
Thank you for pointing out this. Actually it's only required in Layout file. We don't need it in DarkModeToggle file. I fixed it. Sorry for previous mistakes.